packaging/git-evtag.spec.in

```diff
@@ -1,17 +1,15 @@
-%global commit0 74fb3c0814152de193bac65a53b86a08a25ecb79
-%global gittag0 v2016.1
-%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
```

Comment:
in case you use git snapshot - yes, but not when you use released version. also that hardcoded stuff. if you will bump Version tag, you will still download same sources.